src/clr-angular/utils/_helpers.clarity.scss

```diff
@@ -63,6 +63,12 @@ $opp-directions: (
   }
 }
 
+@mixin fixForIE9AndUp() {
+  @media screen and (min-width: 0\0) and (min-resolution: +72dpi) {
```

Comment:
How sure are you that this only targets IE11? Can you comment/link to an explanation of the hack?

Reply:
This is targeting more IE versions ( 9, 10 and 11). Many sources doesnt explain properly how these hacks work.
Meanwhile, i found another hack which seems to have a plausible explanation
``@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {  
   /* specific styles go here */  
}``
This works for 10 and 11 only. It seems like IE10 and 11 (only) support the -ms-high-contrast media query. If the actual IE11 targeting mixin doesnt work, what do you think about changing it with another one?

Reply:
This 
```
@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) { /* specific styles go here */ }
```
was the hack I also saw for targeting IE11. Lets go with that and figure out why the mixin isn't working for you. Can you provide any feedback or compile errors related to the the mixin name?

Reply:
Lets try the recommendation below from @gnomeontherun before we proceed with the hacky solution.